### DISCUSSION TITLE
Bottle support for Intel Macs running Sequoia (macOS 15 x86_64)?

### DISCUSSION THREAD

**Discussion by joshlawton** [Homebrew/discussions#5678] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 70672606c6ac07a5e8f75abeda7b8736e8285dbe
Last commit: 2 days ago
Core tap JSON: 16 Oct 08:27 UTC
Core cask tap JSON: 16 Oct 08:27 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.5 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.5/bin/ruby
CPU: octa-core 64-bit haswell
Clang: 16.0.0 build 1600
Git: 2.39.5 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.0.1-x86_64
CLT: 16.0.0.0.1.1724870825
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hey maintainers -- Could you shed some light on the plan, if any, to support installing formulae from bottles for Intel Macs running Sequoia? The best I could find is [this issue](https://github.com/Homebrew/brew/issues/18356) and [this pull request](https://github.com/Homebrew/brew/pull/18363).

**Comment by SMillerDev** [comment]
> Could you shed some light on the plan, if any, to support installing formulae from bottles for Intel Macs running Sequoia?

https://brew.sh/2024/10/01/homebrew-4.4.0/ mentions: "The most significant changes since 4.3.0 are official macOS Sequoia (15)" so that means that Intel Macs running Sequoia are supported.

**Comment by p-linnane** [comment]
Intel Sequoia uses Sonoma bottles except where there is a need for a dedicated Intel Sequoia bottle (`pkg-config` is one example). We are constrained with our CI resources, and Apple is no longer producing Intel hardware. This is the best way we can support users still on Intel devices, but you should expect to see concessions like this becoming more frequent as we approach Apple dropping support for Intel Macs.

**Reply by Bo98** [reply]
I will add for this that if you feel there's a formula that isn't working with a Sonoma bottle, do open a discussion and we can check. We expect >99% of formulae to work fine on Sequoia using Sonoma bottles.

The only known issue so far is `gcc` which was waiting for a fix upstream that just got released today - we'll look into that again soon.
